pkgs/desktops/xfce/panel-plugins/xfce4-datetime-plugin/default.nix

```diff
@@ -26,6 +26,5 @@ mkXfceDerivation {
 
   meta = with lib; {
     description = "Shows the date and time in the panel, and a calendar appears when you left-click on it";
```

Comment:
```suggestion
    description = "Shows the date and time in the panel, and a calendar appears when you left-click on it";
    maintainers = with maintainers; [ ] ++ teams.xfce.members;
```

Reply:
The maintainer is already set in `mkXfceDerivation`.

Reply:
Maybe it should be possible to add a plugin-specific maintainer,
without being part of `teams.xfce.members` nor responsible for all `mkXfceDerivation` invocations.
Hiding `maintainers` might be a hindrance for future plugin-specific maintainers.

Reply:
Would it work better if `maintainers` is not set in `mkXfceDerivation`, but individually in each package?

Reply:
Yes.
`maintainers` should be set individually for each package (additionally from `mkXfceDerivation` for all plugins).

Reply:
I am thinking about not setting `maintainers` in `mkXfceDerivation`, but only setting it individually in each package (including each plugin). This would be more flexible because the team wouldn't have to be the maintainers for all and every package defined with `mkXfceDerivation` in nixpkgs.

Is that ok?

Reply:
Yes, totally fine with me.

Reply:
Done.